eve/workers/compile_centos_8/Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM centos:7.7.1908
+FROM centos:8.2.2004
```

Comment:
we don't have the exact same one for [the server](https://github.com/scality/hyperdrive), might update it.